examples/resources/credential/cluster_proxy.tf

```diff
@@ -1,13 +1,14 @@
-# Create cluster proxy credential
-resource "tanzu-mission-control_credential" "proxy_cred" {
-  name = "proxy_cred"
+# Create explicit cluster proxy credential
+resource "tanzu-mission-control_credential" "explicit_proxy_cred" {
+  name = "explicit_proxy_cred"
 
   meta {
-    description = "proxy credential"
+    description = "explicit proxy credential"
     labels = {
       "key1" : "value1",
     }
     annotations = {
+      "proxyType" : "explicit",
```

Comment:
If there is no annotation specified it will be considered as explicit proxy, for older proxy credential as well. There is no specific operation in such case

Reply:
@shobha2626 can we add a note in to [credential.md](https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/219/files#diff-4b4bcc84a3b8fb567475215bbec3c11fde0580f3e7da460d1f0cb946e2eeb904) regarding proxy credentials created prior to v1.2.1

Reply:
@vmw-vjn I have captured it in here - https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/f8bca04e22a74d384b1eca2d9894372021b7a659/docs/data-sources/credential.md?plain=1#L40

Reply:
Removing this description as annotation is a common type used across resources.
Instead adding as a comment as suggested.